glide.yaml

```diff
@@ -20,3 +20,5 @@ import:
 - package: github.com/arschles/kubeapp
   subpackages:
   - api
+- package: github.com/docker/docker
+  version: 2b27fe17a1b3fb8472fde96d768fa70996adf201
```

Comment:
right, sorry I wasn't unclear. I'm suggesting just doing a `glide up` and checking in the new `glide.lock` that results, instead of pinning the version in the `glide.yaml`